### PR TITLE
CONFSRVDEV-11453 - Update ApacheDS to version 2.0.0-M15, remove suppo…

### PR DESCRIPTION
**Pull request by duyluongtruong**
Update ApacheDS to version 2.0.0-M15, remove support OpenDJ

### CODE REVIEW

File: ldap-maven-plugin/pom.xml [hunk 1]
```diff
@@ -137,11 +137,11 @@
             <artifactId>server-apacheds</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>com.btmatthews.maven.plugins.ldap</groupId>
-            <artifactId>server-opendj</artifactId>
-            <scope>test</scope>
-        </dependency>
+        <!--        <dependency>-->
```
Comment (duyluongtruong) [inline ldap-maven-plugin/pom.xml]: remove this

File: pom.xml [hunk 2]
```diff
@@ -434,40 +429,96 @@
...
+                <artifactId>apacheds-jdbm1</artifactId>
+                <version>2.0.0-M3</version>
             </dependency>
+
             <dependency>
                 <groupId>org.apache.directory.shared</groupId>
                 <artifactId>shared-ldap</artifactId>
                 <version>0.9.15</version>
             </dependency>
             <dependency>
```
Comment (duyluongtruong) [inline pom.xml:518]: remove this dependency as well

File: ldap-maven-plugin/pom.xml [hunk 3]
```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>com.btmatthews.maven.plugins.ldap</groupId>
         <artifactId>parent</artifactId>
-        <version>1.3.3-SNAPSHOT</version>
+        <version>1.3.3-M01</version>
```
Comment (minhtran83) [inline ldap-maven-plugin/pom.xml]: We should need to update the `public-pom` in order to release this library

File: ldap-maven-plugin/pom.xml (hunk 1 above)
Comment (duyluongtruong) [reply ldap-maven-plugin/pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (duyluongtruong) [reply pom.xml:518]: done

File: ldap-maven-plugin/pom.xml (hunk 3 above)
Comment (duyluongtruong) [inline ldap-maven-plugin/pom.xml]: Should we use Atlassian specific version ?

**Review by minhtran83 — APPROVED** [review]

File: ldap-maven-plugin/pom.xml (hunk 3 above)
Comment (duyluongtruong) [reply ldap-maven-plugin/pom.xml]: Other lib fork does not do that. I'm confusing. I guess I will try to release with this for now. We could always change it later

**Pull request merged**